packages/directory/addon/components/dir-icon.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * {{dir-icon
+ *    value=value
+ * }}
+ */
+import Component from '@ember/component';
+import layout from '../templates/components/dir-icon';
+import { get, computed } from '@ember/object';
+import { A as arr } from '@ember/array';
+import config from 'ember-get-config';
+
+export default Component.extend({
```

Comment:
Do we need this component?
Can we make `fileTypes` and object and do the lookup based on model name of the item in the template directly?

Reply:
For now it should be a component so it can be used in the `cellComponent` field in ember-light-table. I'll remove this component in the next phase.